src/wallet.js

```diff
@@ -523,8 +523,10 @@ const wallet = {
 
       const stopIndex = startIndex + count;
       for (var i=startIndex; i<stopIndex; i++) {
-        // Generate each key from index, encrypt and save
-        const address = this.generateAddress(i);
+        // Generate each key from index, encrypt and save ( if it is not provided pre-calculated )
+        const address = preCalculatedAddresses && preCalculatedAddresses[i]
```

Comment:
I think we should use `lodash.get` here, what do you think?

Reply:
I think this is more a matter of code style: `lodash.get` is way less performant, but this is irrelevant in this case. I'm open to the idea.

What do you think of:
```suggestion
        const address = _.get(preCalculatedAddresses[i], this.generateAddress(i).toString());
```